### PR TITLE
[tests-only][full-ci] report tracings of failing tests only

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Currently, we get the tracing reports of all the tests that have been retried regardless of pass or fail. And with keycloak tracing is enabled which causes all tracing to be reported for all tests.

Screenshot from 2024-09-13 10-02-17

Report only the tracings of failed tests. With this PR, now we should get the tracing reports of failed tests only.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -1829,7 +1829,7 @@ def e2eTestsOnKeycloak(ctx):
                          "BASE_URL_OCIS": "ocis:9200",
                          "HEADLESS": "true",
                          "RETRY": "1",
-                         "REPORT_TRACING": "true",
+                         "REPORT_TRACING": "with-tracing" in ctx.build.title.lower(),
```
Comment (ScharfViktor) [inline .drone.star:1832]: sorry I don't get it. That means we only get tracing for keycloak test is we set `with-tracing` to the PR header?

File: tests/e2e/cucumber/environment/index.ts [hunk 2]
```diff
@@ -145,9 +154,35 @@ AfterAll(async () => {
   if (state.browser) {
     await state.browser.close()
   }
+
+  if (config.reportTracing) {
+    // move failed tracing reports
+    const failedDir = path.dirname(config.tracingReportDir) + '/failed'
```
Comment (ScharfViktor) [inline tests/e2e/cucumber/environment/index.ts:160]: should we change path here?
https://github.com/owncloud/web/pull/11571/files#diff-6a968d4b7901c8526c8d9a86d1f434737dbfe2b00da21a7d3041ed4e3d58d62eR1512-R1513

to `reports/e2e/playwright/tracing/failed`

File: .drone.star (hunk 1 above)
Comment (saw-jan) [reply .drone.star:1832]: we get tracing if there's with-tracing in the title or tracings are enabled if a scenario is retried after failing.

File: tests/e2e/cucumber/environment/index.ts (hunk 2 above)
Comment (saw-jan) [reply tests/e2e/cucumber/environment/index.ts:160]: All failed tracings will be moved to the original path after all tests are executed

Comment (saw-jan) [reply tests/e2e/cucumber/environment/index.ts:160]: So that means there will be no change in the tracing paths at the end

**Review by ScharfViktor — APPROVED** [review]
👍

**Pull request merged**